src/definition/livechat/IPostLivechatRoomSaved.ts

```diff
@@ -0,0 +1,21 @@
+import { IHttp, IModify, IPersistence, IRead } from '../accessors';
+import { AppMethod } from '../metadata';
+import { ILivechatRoom } from './ILivechatRoom';
+
+/**
+ * Handler called after the room's info get saved.
+ */
+export interface IPostLivechatRoomSaved {
+    /**
+     * Handler called *after* the room's info get saved.
+     *
+     * @param data the livechat context data which contains room's info.
+     * @param read An accessor to the environment
+     * @param http An accessor to the outside world
+     * @param persis An accessor to the App's persistence
+     * @param modify An accessor to the modifier
+     */
+    [AppMethod.EXECUTE_POST_LIVECHAT_ROOM_SAVED](
+        context: ILivechatRoom, read: IRead, http: IHttp, persis: IPersistence, modify?: IModify,
```

Comment:
When introducing new methods, you can leave the modify parameter as a non-optional one :)

```suggestion
        context: ILivechatRoom, read: IRead, http: IHttp, persis: IPersistence, modify: IModify,
```